docs/Configuration.md

```diff
@@ -178,12 +178,16 @@ e.g.:
   - path: "src/**/*.cpp"
     start_lines: "3"
     line_count: 11
-    detected_license: "Apache-1.1"
+    detected_license: "GPL-2.0-only"
+    reason: CODE,
```

Comment:
The trailing comma looks wrong here, or? And mustn't "CODE" be quoted?

Reply:
Ping @fviernau to ensure this does not get overlooked despite the PR being merged meanwhile.

Reply:
Good finding, fixing it now in a follow up PR.

Reply:
Quoting code seems not to be necessary. Do you prefer to have the quoting still? ...I could add it to the fix-up PR : https://github.com/heremaps/oss-review-toolkit/pull/1857

Reply:
I'd prefer quoting if that's the way the config would be serialized. Looking at other enums like *project linkage* it seems that enums *do* get quoted usually, so yes, then I'd prefer that.

Reply:
Thanks, makese sense.